Source/SwiftLintFramework/Rules/RuleConfigs/SeverityLevelsConfig.swift

```diff
@@ -10,24 +10,25 @@ import Foundation
 
 public struct SeverityLevelsConfig: RuleConfig, Equatable {
     var warning: Int
-    var error: Int
+    var error: Int?
 
     var params: [RuleParameter<Int>] {
-        return [RuleParameter(severity: .Error, value: error),
+        if let error = error {
+            return [RuleParameter(severity: .Error, value: error),
                 RuleParameter(severity: .Warning, value: warning)]
+        }
+        return [RuleParameter(severity: .Warning, value: warning)]
     }
 
     mutating public func setConfig(config: AnyObject) throws {
         if let config = [Int].arrayOf(config) where !config.isEmpty {
             warning = config[0]
-            if config.count > 1 {
-                error = config[1]
-            }
+            error = (config.count > 1) ? config[1] : nil
         } else if let config = config as? [String: AnyObject] {
             if let warningNumber = config["warning"] as? Int {
                 warning = warningNumber
-            }
-            if let errorNumber = config["error"] as? Int {
+                error = config["error"] as? Int
+            } else if let errorNumber = config["error"] as? Int {
                 error = errorNumber
```

Comment:
I'm pretty sure this can just be replace with:

```swift
if let warningNumber = config["warning"] as? Int {
    warning = warningNumber
}
error = config["error"] as? Int
```

What do you think?